CloudViewConfig/cv/data/config/DataModels.xml

```diff
@@ -151,7 +151,7 @@
         </dmv1:DataModelClass>
         <dmv1:DataModelClass name="3dx" parentClass="document">
             <dmv1:ExpansionParams generateAnalysis="true" generateIndexSchema="true" generatePrefix="true" generateFacet="true" generateMeta="true"/>
-            <dmv1:AlphanumProperty semanticType="metadata" indexInTextField="false" enumerated="false" overrideClassExpansionParams="false" dedicatedField="true" searchable="true" retrievable="true" ramBased="false" facet="false" disjunctiveRefines="false" arity="zero_or_one" name="id">
```

Comment:
Confirm if this value should be changed from false to true !

Reply:
resolved. it's true.